packages/astro/test/fixtures/content-collection-references/src/pages/welcome-data.json.js

```diff
@@ -4,9 +4,7 @@ export async function GET() {
 	const welcomePost = await getEntry('blog', 'welcome');
 
 	if (!welcomePost?.data) {
-		return { 
-			body: { error: 'blog/welcome did not return `data`.' },
-		}
+		return Response.json({ error: 'blog/welcome did not return `data`.' })
```

Comment:
```suggestion
		return Response.json({ error: 'blog/welcome did not return `data`.' }, { status: 404 })
```

It's a fixture so I know it doesn't matter but `Response.json` can return a status as well.